pro/src/pages/Reimbursements/ReimbursementsInvoices/InvoiceTable/InvoiceTable.tsx

```diff
@@ -32,7 +32,7 @@ enum InvoicesOrderedBy {
 }
 
 function sortInvoices(
-  invoices: InvoiceResponseModel[],
+  invoices: InvoiceResponseV2Model[],
```

Comment:
Il ne vaudrait pas mieux utiliser un truc du genre ?
```suggestion
  invoices: InvoiceResponseModel[] | InvoiceResponseV2Model[],
```

Reply:
Je trouve ça plus compliqué de gérer 2 types différents qu'un seul. Surtout que la cible c'est d'utilisé la V2 donc je trouvais ça plus pertinent de simplement transformer `InvoiceResponseModel` -> `InvoiceResponseV2Model`, ça consiste juste à mettre `reimbursementPointName` dans `bankAccountLabel`. 

La solution avec la concatenation des 2 types me semble plus compliqué et moins lisible pour avoir essayer